pkg/cmd/get/packages.go

```diff
@@ -0,0 +1,165 @@
+package get
+
+import (
+	"context"
+	"fmt"
+	"github.com/cnoe-io/idpbuilder/api/v1alpha1"
+	"github.com/cnoe-io/idpbuilder/pkg/build"
+	"github.com/cnoe-io/idpbuilder/pkg/k8s"
+	"github.com/cnoe-io/idpbuilder/pkg/printer"
+	"github.com/cnoe-io/idpbuilder/pkg/types"
+	"github.com/spf13/cobra"
+	"io"
+	"k8s.io/client-go/util/homedir"
+	"os"
+	"path/filepath"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"strconv"
+)
+
+// TODO To be removed when we will merge PR: 442 as duplicate
```

Comment:
https://github.com/cnoe-io/idpbuilder/pull/442/files is now merged so maybe need to go through this again?